frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -833,11 +833,25 @@ export async function getJobs(): Promise<Array<APIJob>> {
     tracingId: job.commandArgs.kwargs.volume_tracing_id,
     annotationId: job.commandArgs.kwargs.annotation_id,
     annotationType: job.commandArgs.kwargs.annotation_type,
-    state: job.celeryInfo.state || "UNKNOWN",
+    state: adaptJobState(job.command, job.celeryInfo.state, job.manualState),
+    manualState: job.manualState,
     createdAt: job.created,
   }));
 }
 
+function adaptJobState(command: string, celeryState: string, manualState: ?string): string {
```

Comment:
```suggestion
function adaptJobState(command: string, celeryState: ?string, manualState: ?string): string {
```

I'd assume this at least, since the (old and new) code explicitly handles a falsy value of `celeryState`. Yet another instance where I would have hoped that flow warns about..